klippy/extras/probe.py

```diff
@@ -311,13 +329,9 @@ def get_lift_speed(self):
     def _move_next(self):
         toolhead = self.printer.lookup_object('toolhead')
         # Lift toolhead
-        speed = self.lift_speed
-        if not self.results:
-            # Use full speed to first probe position
-            speed = self.speed
         curpos = toolhead.get_position()
         curpos[2] = self.horizontal_move_z
-        toolhead.move(curpos, speed)
+        toolhead.move(curpos, self.lift_speed)
```

Comment:
This part of the patch doesn't look correct.  The reason we use the regular moving speed on the first move is for delta printers - the user often starts the probing when the head is very far away from the bed and we want to use the regular move speed (neither lift nor probe) when making the initial move towards the bed.  Also, it seems unrelated to the focus of this PR, so I'd raise in a separate one if a change is needed.

Reply:
Agreed, I didn't realize this was important for deltas. I will undo and fix that. But I will do some testing because I remember that this place was important for some reason while I was searching for the places responsible for my speed observations during intial analysis when probing - I will report on my findings.

Reply:
Obsoleted by refactoring